robot_skills/src/robot_skills/torso.py

```diff
@@ -10,7 +10,7 @@
 
 # TU/e Robotics
 from robot_part import RobotPart
-from .util import concurrent_util
```

Comment:
I kinda like the indication that `.util` is local import. Moveover, I'd like to add the dot-prefix to `.robot_part` as well. 

That is more explicit, and explicit is better than implicit  (try `import this` in your python interpreter)

Reply:
My Pycharm started complaining about it. So therefore I removed it. Without dot works both in terminal and PyCharm. With dot only in terminal.

Reply:
Switch to absolute imports?

Reply:
Thats what we do for all of the other import I think, so then please do absolute imports. Much clearer.